sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -193,8 +193,8 @@ public EventParticipantDto saveEventParticipant(EventParticipantDto dto) {
 		Event event = eventService.getByUuid(eventReferenceDto.getUuid());
 
 		if (!eventJurisdictionChecker.isInJurisdiction(event) && (dto.getRegion() == null || dto.getDistrict() == null)) {
-			dto.setRegion(new RegionReferenceDto(user.getRegion().getUuid()));
-			dto.setDistrict(new DistrictReferenceDto(user.getDistrict().getUuid()));
+			dto.setRegion(new RegionReferenceDto(user.getRegion() != null ? user.getRegion().getUuid() : null));
```

Comment:
should be `user.getRegion() ? new RegionReferenceDto() : null`

Reply:
changed